ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -98,7 +98,7 @@
     - name: Install oc
       copy:
         remote_src: True
-        src: "/tmp/{{ openshift_client_release }}/oc"
+        src: "/tmp/oc"
```

Comment:
The .zip format is used for the mac version of client tools.
I tested the oc client tools for 3.6, 3.7, and 3.9 all of them extracted without a directory being created.
Looks like right thing to do is remove the intermediate directory.